okhttp/src/main/java/feign/okhttp/OkHttpClient.java

```diff
@@ -106,6 +105,9 @@ static Request toOkHttpRequest(feign.Request input) {
 
   private static feign.Response.Body toBody(final ResponseBody input) throws IOException {
     if (input == null || input.contentLength() == 0) {
+      if (input != null) {
+        input.close();
```

Comment:
Seems odd. `input.close()` is in `close()` a few lines below. Why isn't that being called? What happens if `close()` is called before the input stream is read?